librosa/display.py

```diff
@@ -505,15 +522,21 @@ class ChromaSvaraFormatter(mplticker.Formatter):
 
     """
 
-    def __init__(self, Sa=None, mela=None, abbr=True, unicode=True):
+    def __init__(
+        self,
+        Sa: Optional[np.ndarray] = None,
```

Comment:
Sa should be scalar here (midi number in this context)

Reply:
Done